### PR TITLE
FreeWheelAdapter-CCPA support CCPA consent in freeWheelSSPAdapter

### PR DESCRIPTION
**Pull request by xwang202**

### CODE REVIEW

**Comment by xwang202** [comment]
Hi @jsnellbaker , 

I've submitted the changes on CCPA support of FreeWheel SSP Adapter. Please review. 

Thanks,
Xuan

**Comment by xwang202** [comment]
Hi  @jsnellbaker, 

I've updated the tests and docs (https://github.com/prebid/prebid.github.io/pull/1734), please review. 

Thanks,
Xuan

**Comment by xwang202** [comment]
Hi @jsnellbaker,

seems there is a timeout in unit test. can you please help to re-run it?

Thanks,
Xuan

File: test/spec/modules/freewheel-sspBidAdapter_spec.js [hunk 1]
```diff
@@ -86,6 +86,43 @@ describe('freewheelSSP BidAdapter Test', () => {
...
       expect(request.method).to.equal('GET');
     });
+
+    it('should add usp consent to the request', () => {
+      let uspConsentString = '1FW-SSP-uspConsent-';
+      let bidderRequest = {};
+      bidderRequest.uspConsent = uspConsentString;
+
+      const request = spec.buildRequests(bidRequests, bidderRequest);
+      console.log(request.data);
```
Comment (jsnellbaker) [inline test/spec/modules/freewheel-sspBidAdapter_spec.js]: Can you remove this console.log statement please?

File: test/spec/modules/freewheel-sspBidAdapter_spec.js [hunk 2]
```diff
@@ -86,6 +86,43 @@ describe('freewheelSSP BidAdapter Test', () => {
...
+    it('should add gdpr consent to the request', () => {
+      let gdprConsentString = '1FW-SSP-gdprConsent-';
+      let bidderRequest = {
+        'gdprConsent': {
+          'consentString': gdprConsentString
+        }
+      };
+
+      const request = spec.buildRequests(bidRequests, bidderRequest);
+      console.log(request.data);
```
Comment (jsnellbaker) [inline test/spec/modules/freewheel-sspBidAdapter_spec.js]: Same here - can you remove this console.log statement please?

File: test/spec/modules/freewheel-sspBidAdapter_spec.js (hunk 1 above)
Comment (xwang202) [reply test/spec/modules/freewheel-sspBidAdapter_spec.js]: Removed.

File: test/spec/modules/freewheel-sspBidAdapter_spec.js (hunk 2 above)
Comment (xwang202) [reply test/spec/modules/freewheel-sspBidAdapter_spec.js]: Removed.

**Pull request merged**